### ISSUE TITLE
Tutoriel d'intégration de package issus de Pytorch-Geometric

### ISSUE THREAD

**Issue by CharlesGaydon** [IGNF/myria3d#21]
Les deux architectures actuellement proposées suivent globalement la structure de données de [pyg](https://github.com/pyg-team/pytorch_geometric), tels que listés dans [les exemples de la librairie](https://github.com/pyg-team/pytorch_geometric/tree/master/examples).

Éléments importants pour la compatibilité : 
- data.batch (variable attendue dans pyg) est actuellement nommé data.batch_x, et correspond aux indices de batch sub-samplés ; data.batch_y correspond lui au indices non sub-samplés, qui sont utilisés pour extraire les éléments y et pos. Ces noms ne sont pas heureux ; data.batch_x peut être renommé en batch, et batch_x en batch_copy pour clarifier et assurer la compatibilité.
- Le modèle factory qui fournit les architecture PointNet ou RandLaNet n'est pas extensible facilement ; privilégier un chemin direct vers le module en question si possible.
- Les exemples dans pyg retournent parfois des log probabilités et non des logits. La prédiction par fenêtre glissante avec overlap exige quant à elle de travailler sur les logits, ce qui est fait actuellement. Il faut simplement penser à utiliser les logits sans log_softmax en sortie.

Pour valider cette compatibilité, on peut proposer un unique modèle très simple basé sur pyg, par exemple le GCN simple du [README](https://github.com/pyg-team/pytorch_geometric#train-your-own-gnn-model).

**Comment by CharlesGaydon** [comment]
A existé un temps avec PN++. Mais lourd à maintenir si pas utilisé. On privilégiera une version actualisée de RandLaNet codée avec PyG.
